src/components/betting-chips/BettingControls.tsx

```diff
@@ -0,0 +1,78 @@
+import React from "react";
+import { observer } from "mobx-react";
+import { useMainStore } from "../../hooks/useMainStore";
+import { GameStage } from "../../stores/gameStore";
+import { BettingChip, BettingChipsValues } from "./BettingChip";
+
+const bettingChipValues: BettingChipsValues[] = [1, 5, 25, 50, 100];
+
+export const BettingControls = observer(() => {
+  const { game, betweenRoundsReset } = useMainStore();
+  const [bettingChoice, setBettingChoice] = React.useState("");
+
+  const onOptionChange = (e: React.ChangeEvent<HTMLInputElement>) => {
+    setBettingChoice(e.target.value);
+  };
+
+  const handleDeal = () => {
+    if (game.gameStage === GameStage.InitialBet) {
+      game.setGameStage(GameStage.InitialCards);
+    }
+    if (game.gameStage === GameStage.SecondBet) {
+      game.setGameStage(GameStage.CheckForThirdCard);
+    }
+  };
+
+  return (
+    <>
+      {bettingChipValues.map((value) => (
+        <BettingChip key={value} value={value} betOn={bettingChoice} />
+      ))}
+      <button type="button" onClick={handleDeal}>
```

Comment:
This button must be `disabled` after first click. Does it really need to have the **SecondBet** game stage? I check baccarat game and "Deal" button is only clicked once, whenever additional card is required it's being drawn automatically.